File: src/main/java/hu/eenugw/core/helpers/UUIDHelpers.java
package hu.eenugw.core.helpers;

import java.util.UUID;

public class UUIDHelpers {
    public static final String DEFAULT_UUID = "00000000-0000-0000-0000-000000000000";

    public static String generateUUID() {
        return UUID.randomUUID().toString();
    }
}

File: frontend/helpers/uuidHelpers.ts
import { v4 as uuidv4 } from 'uuid';

class UUIDHelpers {
    static getDefaultUUID(): string {
        return '00000000-0000-0000-0000-000000000000';
    }

    static generateUUID(): string {
        return uuidv4();
    }
}

export default UUIDHelpers;
